### PR TITLE
RIA-3155 notifications to parties when appeal reinstated

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-3155

### Change description ###

Notifications to parties when appeal reinstated

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/homeoffice/HomeOfficeReinstateAppealPersonalisation.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                .orElse(null);
+
+        if (state == null) {
+            return null;
+        }
+
+        if (Arrays.asList(
+                State.APPEAL_SUBMITTED,
+                State.PENDING_PAYMENT,
+                state.AWAITING_RESPONDENT_EVIDENCE,
```
Comment (makowalski) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/homeoffice/HomeOfficeReinstateAppealPersonalisation.java:79]: typo, it should be `State`

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**